test/trento/application/usecases/health_summary_service_test.exs

```diff
@@ -47,19 +49,43 @@ defmodule Trento.HealthSummaryServiceTest do
     end
 
     test "should determine health summary for a SAP System" do
-      %{
-        sap_system_id: sap_system_id,
+      %Trento.ClusterReadModel{id: cluster_id} =
+        insert(:cluster, type: ClusterType.hana_scale_up(), health: Health.passing())
+
+      %Trento.HostReadModel{id: host_1_id} =
+        insert(:host, cluster_id: cluster_id, heartbeat: :unknown)
+
+      %Trento.SapSystemReadModel{
+        id: sap_system_id,
         sid: sid
-      } = sap_system_with_cluster_and_hosts()
+      } = insert(:sap_system, health: Health.critical())
+
+      insert(
+        :database_instance_without_host,
+        sap_system_id: sap_system_id,
+        sid: "HDD",
+        host_id: host_1_id,
+        health: Health.warning()
+      )
+
+      insert(
+        :application_instance_without_host,
+        sap_system_id: sap_system_id,
+        sid: sid,
+        host_id: host_1_id,
+        health: Health.critical()
+      )
 
       assert [
                %HealthSummaryDto{
                  id: ^sap_system_id,
                  sid: ^sid,
-                 sapsystem_health: :passing,
-                 database_health: :critical,
-                 clusters_health: :warning,
-                 hosts_health: :unknown
+                 sapsystem_health: :critical,
+                 database_health: :warning,
+                 clusters_health: :passing,
+                 hosts_health: :unknown,
+                 database_id: ^sap_system_id,
```

Comment:
now I bit confused it this is the database id or the sap system id hehe
Because if it is the same, we are sending it twice, in the `id` and in the `database_id`.

Now i'm thinking both of them might have the same id, the difference in the frontend is that we get one with `/sap_systems/id` and the other with `/databases/id`.
Could you double check this?

Reply:
As far as I understood, they are not the same 👀

Reply:
They can but can also be different